### DISCUSSION TITLE
Help make proxy non-standard - external nginx proxy

### DISCUSSION THREAD

**Discussion by IwanSE** [goharbor/harbor#19519] [General]
Hello everybody!

I'm solving a non-standard problem.

I need to make a project proxy - harbor.domain.zone/project-name.

The goal is to configure nginx to perform pull.
We are temporarily unable to perform regular docker pull - `docker pull harbor.domain.zone/project-name/kubectl:latest`

I'm trying to configure it using this guide - [Link](https://blog.smile.eu/en/devops/how-configure-docker-hub-proxy-harbor)
I get the error: 

```bash
docker pull some-name.domain.tech/kubectl:latest
Error response from daemon: unauthorized: unauthorized to access repository: temp-proxy2nexus/kubectl, action: pull: unauthorized to access repository: temp-proxy2nexus/kubectl, action: pull
```
Please help us find a solution together, I think that such a solution will help many!
Sorry for the personal note, but maybe you can help, I will be very grateful!
@Vad1mo, @OrlinVasilev, @AllForNothing, @wy65701436 Maybe you know in which direction to investigate this non-standard case, I will be grateful to any feedback.

**Comment by OrlinVasilev** [comment]
Hi @IwanSE, I'll convert that to discussion as issues are to report problems and discussion is more suitable for this!

**Reply by IwanSE** [reply]
Hi @OrlinVasilev sorry)!

**Comment by OrlinVasilev** [comment]
let's continue here :) thank you @IwanSE

**Reply by IwanSE** [reply]
Let's go!

**Comment by chlins** [comment]
Did you configure the correct credential? and please check the harbor core and registry logs as well.

**Reply by IwanSE** [reply]
Hi @chlins, 

Сredential - is correct.
Project Public

I think that the problem is in the rights check due to the fact that rewrite does not fully work.
Trying fixing.

**Reply by T0LED0** [reply]
@IwanSE  Were you able to get the path rewriting to work?
